targets/targets.json

```diff
@@ -511,6 +511,9 @@
             "SLEEP",
             "SPI"
         ],
+        "macros": [
```

Comment:
why are we adding this macro?

Reply:
As part of linker file updates https://github.com/ARMmbed/mbed-os/pull/9571, it was discovered that these 2 targets needed additional fixes. Hence separate PR was created. 

`LPC11U68` had build issues - to resolve that Stack and heap section was added to linker file and MBED_FAULT_HANDLER_DISABLED was disabled. 

Issues were communicated and resolved over mail with NXP.

Reply:
If this PR is approved, I can cherry-pick commits and apply them on https://github.com/ARMmbed/mbed-os/pull/9571, if approval for 9571 takes more than a week